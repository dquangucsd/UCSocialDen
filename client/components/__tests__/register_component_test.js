import 'react-native';
import React from 'react';
import LoginScreen from '../../app/login';
import { render } from '@testing-library/react-native';

// Mock AsyncStorage

describe('LoginScreen', () => {
  it('renders correctly', () => {
    const { toJSON } = render(<LoginScreen />);
    expect(toJSON()).toMatchSnapshot();
  });
});


