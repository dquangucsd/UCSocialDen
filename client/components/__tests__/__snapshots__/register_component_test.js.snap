// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginScreen renders correctly 1`] = `
<View
  style={
    {
      "backgroundColor": "#fafafa",
      "flex": 1,
    }
  }
>
  <View
    style={
      {
        "alignItems": "center",
        "backgroundColor": "#556ebe",
        "flexDirection": "row",
        "justifyContent": "space-between",
        "padding": 16,
      }
    }
  >
    <Text
      style={
        {
          "color": "#FFFFFF",
          "fontFamily": "Zain",
          "fontSize": 34,
          "fontWeight": "bold",
        }
      }
    >
      UC Social Den
    </Text>
  </View>
  <View
    style={
      {
        "alignItems": "center",
        "backgroundColor": "#fafafa",
        "flex": 1,
      }
    }
  >
    <Text
      style={
        {
          "color": "#556ebe",
          "fontFamily": "Zain",
          "fontSize": 32,
          "fontWeight": "bold",
          "marginBottom": 30,
          "marginTop": 200,
        }
      }
    >
      Welcome to UC Social Den
    </Text>
    <ActivityIndicator
      color="#007AFF"
      size="large"
    />
  </View>
</View>
`;
